packages/fuels-test-helpers/src/signers.rs

```diff
@@ -0,0 +1,26 @@
+use std::net::SocketAddr;
+
+use fuel_core::model::coin::Coin;
+use fuel_tx::UtxoId;
+use fuels_signers::provider::Provider;
+use fuels_signers::LocalWallet;
+
+#[cfg(feature = "fuels-signers")]
+pub async fn setup_test_provider_and_wallet() -> (Provider, LocalWallet) {
+    //  We build only 1 coin with amount TEST_COIN_AMOUNT, empirically determined to be
```

Comment:
where is `TEST_COIN_AMOUNT`

Reply:
yeah I just ctrl+c ctrl+v the `fuels-signers/utils.rs` mostly

Reply:
I switched comment to `DEFAULT_INITIAL_BALANCE`